fem/fe.hpp

```diff
@@ -301,13 +303,21 @@ class FiniteElement
        @param Do   Number of degrees of freedom in the FiniteElement
        @param O    Order/degree of the FiniteElement
        @param F    FunctionSpace type of the FiniteElement
-    */
+       @param VD   Vector dimension for vector-valued bases
+       @param CD   Curl dimension for vector-valued bases
+   */
    FiniteElement(int D, Geometry::Type G, int Do, int O,
-                 int F = FunctionSpace::Pk);
+                 int F = FunctionSpace::Pk, int VD = 0, int CD = 0);
 
    /// Returns the reference space dimension for the finite element
    int GetDim() const { return dim; }
 
+   /// Returns the vector dimension for vector-valued finite elements
+   int GetVDim() const { return vdim; }
+
+   /// Returns the dimension of the curl for vector-valued finite elements
+   int GetCDim() const { return cdim; }
```

Comment:
`GetCurlDim` would be more readable, but I don't have a strong preference.

Reply:
I agree and I also don't have a strong preference. Let's see if anyone else has a preference. I'll be happy to change it if that's the consensus.

Reply:
I prefer `GetCurlDim`